src/ucp/api/ucp.h

```diff
@@ -488,7 +488,7 @@ enum ucp_send_am_flags {
                                               for AM sends. */
     UCP_AM_SEND_RNDV       = UCS_BIT(2), /**< Force UCP to use only rendezvous protocol
                                               for AM sends. */
-    UCP_AM_SEND_FETCH_DATA = UCS_BIT(3)  /**< Tells @ref ucp_am_send_nbx to send
+    UCP_AM_SEND_GET_REPLY  = UCS_BIT(3)  /**< Tells @ref ucp_am_send_nbx to send
```

Comment:
write which fields in request_param have to be specified (reply buffer, reply count)

Reply:
fixed